### PR TITLE
Feat(database)

### PR DESCRIPTION
**Pull request by SamTV12345**
### Check List

- [x] I have formatted using `npx turbo format`
- [x] I have checked the types using `npx turbo type-check`
- [x] I have tested the build using `npx turbo build`
- [ ] I have covered the code with tests if necessary

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -48,3 +48,8 @@ next-env.d.ts
 # Production configuration file
 /settings.json
 storybook-static/
+
+
+# Var files
+var/dirty.db
+
```
Comment (AugustinMauroy) [inline .gitignore]:
```suggestion

# Var files
var/dirty.db

```

File: app/api/group/create/route.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {createGroup} from '@/service/pads/GroupManager';
+import { APIResponse } from '@/types/APIResponse';
+import { NextResponse } from "next/server";
```
Comment (AugustinMauroy) [inline app/api/group/create/route.ts]:
```suggestion
import { NextResponse } from "next/server";
import {createGroup} from '@/service/pads/GroupManager';
import { APIResponse } from '@/types/APIResponse';
```


put node internal, npm package, our code

File: app/api/group/create/route.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  const response = {
+    code: 0,
+    data: {
+      groupId
+    },
+    message: 'ok'
+  } satisfies APIResponse;
+
+  return NextResponse.json(response);
```
Comment (AugustinMauroy) [inline app/api/group/create/route.ts]:
```suggestion
   return Response.json(response, { status: 200 });
```

[mdn ref](https://developer.mozilla.org/en-US/docs/Web/API/Response)

File: backend/Setting.ts [hunk 4]
```diff
@@ -30,10 +29,11 @@
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
-import pinoLogger from 'pino';
 import { SettingsObj } from '@/types/SettingsObject';
 import { findEtherpadRoot, makeAbsolute } from '@/utils/backend/AbsolutePaths';
 import { argvP } from '@/utils/backend/CLI';
+import log4js from 'log4js';
```
Comment (AugustinMauroy) [inline backend/Setting.ts]: why this step backwards?

File: next.config.mjs [hunk 5]
```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
+
+
+
```
Comment (AugustinMauroy) [inline next.config.mjs]:
```suggestion
```

**Review by AugustinMauroy — COMMENTED** [review]
I thinks it's in WIP but here small review

File: backend/Setting.ts (hunk 4 above)
Comment (SamTV12345) [reply backend/Setting.ts]: The problem is that pino is not encoding correctly the vercel messages. So I simply get random characters printed on Windows.

**Comment by SamTV12345** [comment]
@AugustinMauroy Could you check the source code out. I am not really familiar with Next. It seems like it is importing the files again instead of just referencing the current objects. Every request takes currently around 5 seconds.

File: backend/Setting.ts (hunk 4 above)
Comment (AugustinMauroy) [reply backend/Setting.ts]: Ho sad. And why we have Vercel messages ?

Comment (SamTV12345) [reply backend/Setting.ts]: It logs something like this: ✓ Compiled /api/group in 4.6s . I guess the problem is the checkmark as a Unicode character.

**Pull request merged**